geode/dispatcher.py

```diff
@@ -205,13 +206,16 @@ def __init__(self, config=None):
 
         return res
 
-    async def distance_pairs(self, origins, destinations, max_meters=MAX_METERS, sem=None, session=None, provider=None):
+    async def distance_pairs(self, origins, destinations, max_meters=MAX_METERS, sem=None, session=None, provider=None, return_inverse=False):
         sem = sem or asyncio.BoundedSemaphore(MAX_REQUESTS)
 
         origins = origins.round(4)
         destinations = destinations.round(4)
+        locs, inv = np.unique(np.hstack((origins, destinations)), axis=0, return_inverse=True)
```

Comment:
If we're going to shadow the params, should we pass what's coming from up top?

Reply:
not sure what you mean by this